assets/pipeline-slack-notification-lambda/index.py

```diff
@@ -267,17 +284,16 @@ def handler(event, context):
     ]
 
     slack_message = {
-        "channel": SLACK_CHANNEL,
         "attachments": attachments,
-        "username": "Liflig CDK Pipelines",
-        "icon_emoji": ":traffic_light:",
     }
 
-    req = Request(SLACK_URL, json.dumps(slack_message).encode("utf-8"))
+    slack_url = get_secret(SLACK_URL_SECRET_NAME)
+
+    req = Request(slack_url, json.dumps(slack_message).encode("utf-8"))
+    print(f"Message posted to Slack URL {get_masked_slack_webhook_url(slack_url)}")
```

Comment:
nit: The request hasn't actually been made when you print this, so I suggest a different phrasing
```suggestion
    print(f"Posting message to Slack URL {get_masked_slack_webhook_url(slack_url)}")
```